presto-main/src/main/java/com/facebook/presto/util/DateTimeUtils.java

```diff
@@ -548,4 +552,52 @@ else if (parsePos < 0) {
             return ~bestInvalidPos;
         }
     }
+
+    public static SqlTimestamp sqlTimestampOf(int year,
+            int monthOfYear,
+            int dayOfMonth,
+            int hourOfDay,
+            int minuteOfHour,
+            int secondOfMinute,
+            int millisOfSecond,
+            DateTimeZone baseZone,
+            TimeZoneKey timestampZone,
```

Comment:
If I'm not lost, this was about `...#sqlTimestampOf(int, int, int, int, int, int, int, org.joda.time.DateTimeZone, com.facebook.presto.spi.type.TimeZoneKey, com.facebook.presto.spi.ConnectorSession)`

* this is test-only (now)
* i looked briefly through usages, and it seems `timestampZone = session.getTimeZoneKey` in all cases.. didn't check for sure, though